ReadASCIIApp/src/ReadASCII.h

```diff
@@ -43,37 +38,62 @@ class ReadASCII : public asynPortDriver
     int P_StepsPerMin; //float
     int P_CurTemp; //float
     int P_Target; //float
-    int P_SPOut; //float
     int P_SPRBV; //float
 
-    int P_P; //float
-    int P_I; //float
-    int P_D; //float
-    int P_MaxHeat; //float
+	int* lastParam;
```

Comment:
The version of asyn we are currently using does not need to know the number of parameters, so the `lastParam` logic can be removed